select_statements.sql

```diff
@@ -123,27 +94,27 @@ WHERE damage.repairedDate is NULL;
 -- Lägg till kampanj
 
 -- Ta fram alla fakturor under en viss period
-SELECT invoice.invoiceNumber, invoice.bookingNumber, invoice.invoiceSum, invoice.datum, invoice.dueDate, invoice.paid
-FROM invoice
-LEFT JOIN booking ON invoice.bookingNumber = booking.bookingNumber
-WHERE invoice.datum BETWEEN DATE() AND DATE();
+SELECT green_rental.invoice.invoiceNumber, green_rental.invoice.bookingNumber, green_rental.invoice.invoiceSum, green_rental.invoice.invoiceDate, green_rental.invoice.dueDate, green_rental.invoice.paid
```

Comment:
I removed `green_rental.` from all statements to make them easier to read in the report.... etc :D